src/export/ValueSetExporter.ts

```diff
@@ -176,7 +184,9 @@ export class ValueSetExporter {
               this.addConceptComposeElement(composeElement, valueSet.compose.include);
             }
           } else {
-            valueSet.compose.include.push(composeElement);
+            if (composeElement.valueSet?.length != 0) {
+              valueSet.compose.include.push(composeElement);
+            }
```

Comment:
I think this condition might be a little too restrictive. Consider the following FSH:
```
ValueSet: SomeVS
* include codes from system http://example.org/SomeCS and valueset SomeVS
```
In this case, the `composeElement.valueSet` will be an empty array, so `composeElement` will not be added to the value set. But, `composeElement.system` has a value, so I think that `composeElement` should be added.